### ISSUE TITLE
iOS 15.5 support file

### ISSUE THREAD

**Issue by justdan0227** [filsv/iOSDeviceSupport#122]
Anyone know where to get  iOS 15.5 support files

**Comment by brand2021** [comment]
15.5 support files not able to find

**Comment by kirinzer** [comment]
how about now

**Comment by xtyHY** [comment]
The new Xcode 13.4 not include the 15.5 support files. 
If you had install the 15.4 support file or use the Xcode 13.3, you can run directly.

**Comment by aelarassi** [comment]
> [15.5.zip](https://github.com/filsv/iOSDeviceSupport/files/8717283/15.5.zip)

Doesn't work with me!

Xcode : Version 13.2.1 (13C100)
Device : iPhone 7
OS : 15.5 (19F77)

**Comment by hoangquockhanh** [comment]
[15.5.zip](https://github.com/filsv/iOSDeviceSupport/files/8725199/15.5.zip)
I get it from xcode 13.4

**Comment by copypasteho** [comment]
You guys are overthinking this. Just duplicate the 15.4 folder and rename it to 15.5 . Confirmed working on Xcode 12.4 under Catalina . This method can be hit or miss.

**Comment by Python3-8** [comment]
> You guys are overthinking this. Just duplicate the 15.4 folder and rename it to 15.5 . Confirmed working on Xcode 12.4 under Catalina . This method can be hit or miss.

Does this still work? If it does, thank god, my iPhone runs iOS 15.5 and I have Xcode 12.1 on macOS Catalina. Would I need to upgrade to Xcode 12.4 or does it work on 12.1 as well?

**Comment by aelarassi** [comment]
> > You guys are overthinking this. Just duplicate the 15.4 folder and rename it to 15.5 . Confirmed working on Xcode 12.4 under Catalina . This method can be hit or miss.
> 
> Does this still work? If it does, thank god, my iPhone runs iOS 15.5 and I have Xcode 12.1 on macOS Catalina. Would I need to upgrade to Xcode 12.4 or does it work on 12.1 as well?

https://github.com/filsv/iOSDeviceSupport/issues/122#issuecomment-1131371853

**Comment by Python3-8** [comment]
> > > You guys are overthinking this. Just duplicate the 15.4 folder and rename it to 15.5 . Confirmed working on Xcode 12.4 under Catalina . This method can be hit or miss.
> > 
> > 
> > Does this still work? If it does, thank god, my iPhone runs iOS 15.5 and I have Xcode 12.1 on macOS Catalina. Would I need to upgrade to Xcode 12.4 or does it work on 12.1 as well?
> 
> [#122 (comment)](https://github.com/filsv/iOSDeviceSupport/issues/122#issuecomment-1131371853)

Alright, thanks, I'll use that, but do you think I should upgrade to Xcode 12.4? Is it going to make anything better?

**Comment by Python3-8** [comment]
@aelarassi Lol I would appreciate a response

**Comment by Python3-8** [comment]
> [15.5.zip](https://github.com/filsv/iOSDeviceSupport/files/8725199/15.5.zip) I get it from xcode 13.4

Alright, adding to this answer, I was able to get this to work, but with some problems. On my first time trying this, it said it encountered an error while trying to prepare my iPhone for development. All I had to do to fix that was unpair the device, reboot it while restarting Xcode, and repair the device. Then, I got an error saying it was unable to install my app and that the code signature version was no longer supported. To fix this I just followed this https://stackoverflow.com/a/70369149 answer, and that's all! It works!
